libcloud/compute/drivers/kubevirt.py

```diff
@@ -391,63 +772,162 @@ def create_node(
                     )
                     raise KeyError(msg)
 
+                claim_name = disk["volume_spec"]["claim_name"]
+
+                if claim_name not in self.ex_list_persistent_volume_claims(namespace=namespace):
+                    if (
+                        "size" not in disk["volume_spec"]
+                        or "storage_class_name" not in disk["volume_spec"]
+                    ):
+                        msg = (
```

Comment:
Do we have test cases which cover this scenario + all other regular + edge cases?